src/server/ApiRoutes/Hold.js

```diff
@@ -295,7 +295,7 @@ function newHoldRequestServer(req, res, next) {
     (bibResponseError) => {
       res.locals.data.Store = {
         bib: {},
-        searchKeywords: '',
+        searchKeywords: req.query.searchKeywords || '',
```

Comment:
This is the server side handling for search keywords query. Not quite related to the specific message, but is part of the general error handling methods.